numberformat_proposed.html

```diff
@@ -143,11 +141,15 @@ <h1>PartitionNumberPattern ( _numberFormat_, _x_ )</h1>
       </p>
 
       <emu-alg>
-        1. If _x_ is not *NaN* and _x_ < 0, then
-          1. Let _x_ be -_x_.
-          1. Let _pattern_ be _numberFormat_.[[NegativePattern]].
+        1. TODO: Deal with notation here.
```

Comment:
I suppose this TODO refers to CompactDisplay and SignDisplay as well?

Reply:
Yes.  Thanks.  Working on these now.